docs/_includes/head.html

```diff
@@ -2,28 +2,32 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=Edge">
 
-  {% unless site.plugins contains "jekyll-seo-tag" %}
-    <title>{% if page.layout == "i18n_page" %}{% t page.title %}{% else %}{{ page.title }}{% endif %} - {% t "site.title"  %}</title>
-
-    {% if page.description %}
-      <meta name="Description" content="{{ page.description }}">
-    {% endif %}
-  {% endunless %}
+  {% if page.seo_title %}
+    <title>{% t page.seo_title %}</title>
+  {% elsif page.title %}
+    <title>{% t site.name %}: {{ page.title }}</title>
+  {% else %}
+    <title>{% t site.title %}</title>
+  {% endif %}
+  
+  {% if page.seo_description %}
+    <meta name="description" content="{% t page.seo_description %}">
+  {% endif %}
```

Comment:
Previously there was a distinction between the default page layout, and the translated layout (`i18n_page`), where only the latter used the translation plugin to fill in strings. This meant that if you weren't working with page that was translated, you didn't put any strings in the translations file. This seems to be a change in that behavior in the SEO titles and descriptions always need to be in the translations file.

So one question is, should the SEO information change with the language? If so, that's fine. I'm just not sure.

It also seems a bit strange that I need both the `title` and the `seo_title` now (in different files for a `default` layout) and the `title` is only used in the nav menu. Can we combine these to be the same field?

Reply:
So I guess I'd say in a dream world you'd want all the content to be translatable. So it's good that the way it is here supports that.

I'm definitely down to continue having `layout: default` available for pages that don't have translations and making it so that it does not require making edits to the translation files. I just tested and that works now but it does break if you try something like `seo_title: "Sample Page Title"` or `seo_description: "Sample page description."`. Maybe I'm missing something obvious but I couldn't find a way to show the non-translated string if it's there in the template. Any chance you have any ideas? I'll keep digging but figured I'd ask.

On the `title` and `seo_title` pieces I do think they should be separate. For the most part, the SEO titles are longer or more descriptive than the titles shown in the navigation. And the way I set it up it will pull in the `title` if there isn't a specific `seo_title` created.